docs/advanced-topics/messaging-system.md

```diff
@@ -1,8 +1,14 @@
 ---
 id: messaging-system
 title: Messaging System and RPCs
+description: An introdution to the messaging system in MLAPI covering RPC's and Custom Messages
```

Comment:
Just wondering why the 0.1.0 docs for messaging-system don't use this description and content? I'm pretty sure that this should apply for both develop and the 0.1.0 version.

Reply:
I haven't put it in the develop yet, we are updating pages for accessibility as we touch them. so the four files I have touched here have the changes added. most of the other pages don't. Once this is approved then I will make the changes to develop as well. I will chat with Lori to see the best way to do this.

Reply:
You need to update both at the same time when updating any content. If it affects both, update in both. :)

Reply:
don e